### PR TITLE
[CI][UMT] introduce separate migration debian and docker

### PR DESCRIPTION
**Pull request by dkijania**
Introduce separate debian and docker for migration tooling

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: scripts/deb-builder-helpers.sh [hunk 1]
```diff
@@ -328,14 +328,32 @@ build_archive_deb () {
...
   cp ./default/src/app/swap_bad_balances/swap_bad_balances.exe "${BUILDDIR}/usr/local/bin/mina-swap-bad-balances"
 
   build_deb "$ARCHIVE_DEB"
 
+}
+
+##################################### ARCHIVE PACKAGE ##########################################
+build_archive_migration_deb () {
+
+  ARCHIVE_MIGRATION_DEB=mina${DEB_SUFFIX}
```
Comment (emberian) [inline scripts/deb-builder-helpers.sh:343]:
```suggestion
  ARCHIVE_MIGRATION_DEB=mina-archive-migration${DEB_SUFFIX}
```

?

**Review by emberian — APPROVED** [review]

**Comment by dkijania** [comment]
I had to also fix suffix in docker

**Comment by dkijania** [comment]
!ci-build-me

**Review by shimkiv — APPROVED** [review]
LGTM

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
Resolves one of issues :

> Bug: parallel installs clobber the same file ([discord.com/channels/484437221055922177/1204059560684552253/1212477934980440156](https://discord.com/channels/484437221055922177/1204059560684552253/1212477934980440156)). (Probably document not to do that?)

In https://github.com/MinaProtocol/mina/issues/15262

**Comment by dkijania** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!approved-for-mainnet

**Review by nholland94 — APPROVED** [review]

**Pull request merged**